openreview/arr/helpers.py

```diff
@@ -1467,19 +1466,24 @@ def flag_submission(
     short_name = domain.get_content_value('subtitle')
     forum = client.get_note(id=edit.note.forum, details='replies')
 
-    ethics_flag_field = list(flagging_info['ethics_flag_field'].keys())[0]
-    violation_fields = list(flagging_info['violation_fields'].keys())
-    violation_default = flagging_info['violation_fields']
-    ethics_flag_default = flagging_info['ethics_flag_field'][ethics_flag_field]
-    reply_name = flagging_info['reply_name']
-
-    ethics_flag_edits = client.get_note_edits(note_id=edit.note.forum, invitation=f"{venue_id}/-/Ethics_Review_Flag")
-    dsv_flag_edits = client.get_note_edits(note_id=edit.note.forum, invitation=f"{venue_id}/-/Desk_Reject_Verification_Flag")
+    ethics_flag_default = 'No'
+    ethics_flag_fields = {
+        'Review': 'needs_ethics_review',
+        'Checklist': 'need_ethics_review'
+    }
+    violation_fields = {
+        'Checklist': {
+            'appropriateness': 'Yes',
+            'formatting': 'Yes',
+            'length': 'Yes',
+            'anonymity': 'Yes',
+            'responsible_checklist': 'Yes',
+            'limitations': 'Yes'
+        }
+    }
 
-    dsv_flagged = forum.content.get('flagged_for_desk_reject_verification', {}).get('value')
-    ethics_flagged = forum.content.get('flagged_for_ethics_review', {}).get('value')
-    has_ethic_flag_history = len(ethics_flag_edits) > 0
-    has_dsv_flag_history = len(dsv_flag_edits) > 0
+    dsv_flagged = forum.content.get('flagged_for_desk_reject_verification', {}).get('value', False)
```

Comment:
do we need a default value here? I'm not sure if we send a message the is paper is unflagged but it was never flagged.

Reply:
Action is only taken on status change, so if this value is False and none of the checklists are flag the submission then no message is sent